### ISSUE TITLE
Fixme regression in 2.3.0.dev1

### ISSUE THREAD

**Issue by scop** [PyCQA/pylint#2707]
Some fixme improvements from #2320 have apparently been reversed in 2.3.0.dev1 (haven't looked into details, just observing the results). Here's a test file that should not trigger any fixme W0511's:

```python
# Todoist API: What is this task about?
# Todoist API: Look up a task's due date
# Todoist API: Look up a Project/Label/Task ID
# Todoist API: Fetch all labels
# Todoist API: "Name" value
# Todoist API: Get a task's priority
# Todoist API: Look up the Project ID a Task belongs to
# Todoist API: Fetch all Projects
# Todoist API: Fetch all Tasks
            # Todoist's time format; strptime has to be used.
```

pylint 2.2.2 and 2.3.0.dev0 were fine (silent), 2.3.0.dev1 triggers for all of these. (The lines are from https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/calendar/todoist.py)
